qutebrowser/config/configdata.py

```diff
@@ -1125,6 +1124,14 @@ def data(readonly=False):
              SettingValue(typ.QssColor(), 'black'),
              "Background color of the statusbar."),
 
+            ('statusbar.fg.private',
+             SettingValue(typ.QssColor(), '${statusbar.fg}'),
+             "Foreground color of the statusbar in private browsing mode."),
+
+            ('statusbar.bg.private',
+             SettingValue(typ.QssColor(), 'grey'),  # FIXME darker color
```

Comment:
I doubt you'll see the difference on all monitors - what do you think about something like `#444444`? That seems like a clear enough difference, and still all the other colors used are readable.

As for `statusbar.bg.command.private` - I wonder if we really need a different setting for it, or if `.private` should just always trump `.command`...